apps/mobile/src/app/(home)/developer-console/index.tsx

```diff
@@ -119,12 +119,18 @@ export default function DeveloperConsoleScreen() {
         <PressableListItem title={t`Drawer`} />
         <PressableListItem title={t`Page`} />
       </ScrollView>
-      <ApproverSheet
-        fingerprint={wallets.list[0] ? wallets.list[0].fingerprint : ''}
-        accountIndex={0}
-        message={getAddressesMessage}
-        sendResult={() => setGetAddressesMessage(null)}
-      />
+      {wallets.list[0]?.fingerprint && (
```

Comment:
Was curious about our wallet state yest working on the app security level, should we expose something from the wallet read state that is the `wallets.list[0]`, I've seen it repeated in multiple places.

Reply:
I don't think we should really ever be doing `wallets.list[0]`. Only reason we are here is for testing purposes.

Reply:
Got it, thanks for clarifying, I missed this was just in the dev console. 👍